kword/src/commonMain/kotlin/com/mirego/trikot/kword/KWord.kt

```diff
@@ -42,8 +42,11 @@ object KWord : I18N {
     }
 
     override fun t(key: KWordKey, count: Int, vararg arguments: String): String {
-        // TODO: implement zero, one, other
-        return t(key)
+        return when (count) {
+            0 -> t(key)
```

Comment:
It's not that intuitive that the provided key should be the zero value. Maybe rename the parameter to keyOfZero and add some doc?

On a side note I really don't like the fact that we are loosing the type-safety of KWordKey values here 🤔 
I'm pretty sure it would crash at runtime if the _one or _many key is removed from the translation file.

Reply:
Maybe we could ask for all the keys in the method signature, then it would solve both issues (clarity + compile time validation) at the same time?